lib/myxql/protocol.ex

```diff
@@ -331,6 +332,12 @@ defmodule MyXQL.Protocol do
     {:halt, decode_ok_packet_body(rest)}
   end
 
+  def decode_com_query_response(<<0xFB, rest::binary>>, "", :initial) do
+    {filename, _remaining} = take_string_nul(rest)
```

Comment:
Are we sure we can discard the rest here? Or should we assert it is an empty string as well? Or `nul` even there?

Reply:
According [here](https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_com_query_response_local_infile_data.html). The LOCAL INFILE request packet is
- 1 byte: 0xFB and
- N bytes: the filename as a NUL-terminated string

So after parsing the filename, there should be nothing left in the packet